package/smt.spec

```diff
@@ -209,13 +209,20 @@ EOT
     fi
 fi
 
+/usr/lib/SMT/bin/reschedule-sync.sh
+
 %preun
 %service_del_preun smt-schema-upgrade.service
 %service_del_preun smt.service
 %service_del_preun smt.target
 
 # no postun service handling for target or schema-upgrade, we don't want them to be restarted on upgrade
 %postun
+
+if [ -e /etc/cron.d/novell.com-smt-randomized ]; then
+    rm /etc/cron.d/novell.com-smt-randomized
+fi
```

Comment:
I am not sure about this here: AFAIK postun is executed also with every package update. If I understand the code correct this would trigger a re-schedule every update.
Please check and I think there is a way to limit it on only de-installation by checking `$1` (See also http://meinit.nl/rpm-spec-prepostpreunpostun-argument-values)

It also seems that on update `%post` of the new RPM is called first and finally `%postun` from the old RPM.
This would mean, that the old RPM would remove  /etc/cron.d/novell.com-smt-randomized after it got created.

Reply:
That's right, I'll add the `$1` check for `%postun`.

Reply:
I've replaced `%postun` with `%ghost`, it seems more straightforward than `$1` checks. Thanks for the suggestion.